webui/src/pages/bug/CommentForm.graphql

```diff
@@ -1,5 +1,7 @@
 mutation AddComment($input: AddCommentInput!) {
   addComment(input: $input) {
-    operation { id }
+    operation {
+      id
+    }
```

Comment:
Is this reformat intentional?

Reply:
I just ran `make lint-fix` so this is whatever prettify decided to do.